ai_flow/common/util/db_util/session.py

```diff
@@ -80,19 +80,21 @@ def create_sqlalchemy_engine(db_uri):
                                     **pool_kwargs)
 
 
-def create_session(db_uri=None):
+def create_session(db_uri=None, db_engine=None):
```

Comment:
What's the purpose of adding param? If `db_engine` is set, the `db_uri` is ignored ?